Formula/autorest.rb

```diff
@@ -1,8 +1,10 @@
+require "language/node"
+
 class Autorest < Formula
   desc "Swagger (OpenAPI) Specification code generator"
   homepage "https://github.com/Azure/autorest"
-  url "https://api.nuget.org/packages/autorest.0.17.3.nupkg"
-  sha256 "b3f5b67ae1a8aa4f0fd6cf1e51df27ea1867f0c845dbb13c1c608b148bd86296"
+  url "https://registry.npmjs.org/autorest/-/autorest-2.0.4283.tgz"
```

Comment:
Wouldn’t it be more transparent to download from GitHub instead of npmjs.org?

Reply:
I think I got it as an example from another formula. I guess if it's node it's ok to get from npm. One of the benefits I see in it, that we'll not miss release due to upstream have not tagged it in git.

Reply:
One of the downsides would be that if the author were to catch a piece of malware that steals their token and pushes itself to NPM only, we’d have no way to tell.

Reply:
Good point. `2.0.4222` is the last release on github from 12 Dec 2017 though.

Reply:
Fair enough. Thanks for checking!